packages/describe-package/package.json

```diff
@@ -64,7 +61,6 @@
   },
   "dependencies": {
     "@typescript/vfs": "^1.3.5",
```

Comment:
How many of these do we need in `dependencies`, sorry some of them might have been from me.. Can you do a quick pass to see which ones are absolutely required for runtime.

Reply:
Yeah good questions.

I've raised an issue to remove `cross-fetch` entirely when we can safely upgrade engines to node 18.

Over in #100 I've actually made a few changes to the bundle. I'll review this over on that branch so I'm not stepping on my own toes...

Reply:
So for what it's worth I think most of these should still be considered dependencies.

I've moved `tsup` because it's clearly a dev dep, and `threads` because it's bundled into the web worker build. The rest I think are appropriately `dependencies` (at least for now)

Reply:
To be sure, this good to go then?

Reply:
This is good to go :+1: